lib/tasks/default.rake

```diff
@@ -427,6 +427,62 @@ if Rails.env.development?
   end
 end
 
+require 'open-uri'
+# Returns "true" if okay
+# rubocop:disable Metrics/MethodLength
+def validate_one_link(link)
```

Comment:
I agree about the question mark, but that is an awful long name. Maybe something simpler like link_okay?